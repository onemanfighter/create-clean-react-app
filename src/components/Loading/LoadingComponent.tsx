import { Flex, Spinner } from '@chakra-ui/react';

const LoadingComponent = () => {
  return (
    <Flex height={'100vh'} justifyContent={'center'} alignItems={'center'}>
      <Spinner
        thickness="4px"
        speed="0.65s"
        emptyColor="primary"
        color="blue.500"
        size="xl"
      />
    </Flex>
  );
};

export default LoadingComponent;
