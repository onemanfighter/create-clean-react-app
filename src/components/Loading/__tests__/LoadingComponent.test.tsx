import { render } from '@testing-library/react';
import LoadingComponent from '../LoadingComponent';

describe('Loading Component', () => {
  it('should render correctly', () => {
    const { container } = render(<LoadingComponent />);

    expect(container).toMatchSnapshot();
  });
});
