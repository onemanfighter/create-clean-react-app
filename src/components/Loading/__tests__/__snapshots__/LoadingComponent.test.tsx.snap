// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Component should render correctly 1`] = `
<div>
  <div
    class="css-169qivt"
  >
    <div
      class="chakra-spinner css-1nkt52r"
    >
      <span
        class="css-8b45rq"
      >
        Loading...
      </span>
    </div>
  </div>
</div>
`;
